.gitlab/e2e/e2e.yml

```diff
@@ -217,6 +217,26 @@ new-e2e-fips-compliance-test:
   variables:
     TARGETS: ./tests/fips-compliance
     TEAM: agent-runtimes
+  parallel:
+    matrix:
+      - EXTRA_PARAMS: --run "TestFIPSCiphersLinuxSuite$"
+      - EXTRA_PARAMS: --run "TestLinuxFIPSComplianceSuite$"
+
+new-e2e-windows-fips-compliance-test:
+  extends: .new_e2e_template
+  needs:
+    - !reference [.needs_new_e2e_template]
+    - qa_agent_fips
+    - deploy_windows_testing-a7-fips
+  rules:
+    - !reference [.on_asc_or_e2e_changes]
```

Comment:
Gitlab is failing with: `!reference [".on_asc_or_e2e_changes"] could not be found`.

Because the Agent Shared Components rename PR: https://github.com/DataDog/datadog-agent/pull/33477 renamed this rule to `.on_arun_or_e2e_changes`.